core/core-dotspacemacs.el

```diff
@@ -60,7 +60,7 @@ or `spacemacs'.")
 to compile Emacs 27 from source following the instructions in file
 EXPERIMENTAL.org at to root of the git repository.")
 
-(defvar dotspacemacs-emacs-pdumper-executable-file "emacs"
+(defvar dotspacemacs-emacs-pdumper-executable-file "emacs-27.0.50"
```

Comment:
When I build emacs from source I get `emacs-27.0.50.1`... so I would rather not have this change as it is brittle

Reply:
Yep, you can change it to whatever you like. It appears in `.spacemacs`. That is just the default value. I changed it on purpose because people may have a stable Emacs 25 build and a test build for Emacs 26 or 27. So, going this way will specifically pick up version 27 because pdumper isn't there in previous versions.